devops/infrastructure/bash-config.cfg

```diff
@@ -0,0 +1,25 @@
+#### PARAMETERS USED BY AWS
+
+STACK_NAME=joystream-node
+REGION=us-east-1
+CLI_PROFILE=joystream-user
+KEY_PATH="/Users/anuj/Joystream/anuj-key.pem"
+AWS_KEY_PAIR_NAME="anuj-key"
+EC2_INSTANCE_TYPE=t2.xlarge
+
+# If true will create another instance to generate chain spec file and keys
+CREATE_ADMIN_SERVER=true
+
+# Set a prebuilt AMI if required
+EC2_AMI_ID="ami-064f99551235fb1ac"
```

Comment:
I tried setting this to an empty string `EC2_AMI_ID=""` but cloud formation failed with error:
`The image ID '' is not valid. The expected format is ami-xxxxxxxx or ami-xxxxxxxxxxxxxxxxx. `
If I don't want a pre-built image should it just be unset `EC2_AMI_ID=`

Reply:
I tried with empty variable, same result. So I wasn't really able to test the actual build from source unless I remove the "EC2AMI" from the parameters-overrides to cloudformation command.

Reply:
Resolved. Thanks for catching this.